app/api/diagnostic/route.ts

```
@@ -158,24 +185,30 @@
+      return NextResponse.json({ error: "Invalid action" }, { status: 400 });
     }
 
-    const { data: { user } } = await supabase.auth.getUser(); // user can be null
+    const {
+      data: { user },
+    } = await supabase.auth.getUser(); // user can be null
 
     switch (action) {
-      case 'start':
+      case "start":
         const validatedData = validateStartRequest(data);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)